csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/FailFastMetaDataValidator.scala

```diff
@@ -60,16 +60,10 @@ trait FailFastMetaDataValidator extends MetaDataValidator {
     validateRows()
   }
 
-  private def validateRow(row: Row, schema: Schema): MetaDataValidation[Any] = {
```

Comment:
@valydia Ah yes I see, sorry you are correct. I was looking at the wrong context. This bit looks fine.